### PR TITLE
[TEST] New NOTE section for BYT

### PR DESCRIPTION
**Pull request by ktrzcinx**

### CODE REVIEW

File: rimage/elf.c [hunk 1]
```diff
@@ -470,11 +470,11 @@ int elf_validate_modules(struct image *image)
...
 }
 
-int elf_find_section(struct image *image, struct module *module,
+int elf_find_section(const struct image *image, const struct module *module,
 		     const char *name)
 {
-	Elf32_Ehdr *hdr = &module->hdr;
-	Elf32_Shdr *section, *s;
+	const Elf32_Ehdr *hdr = &module->hdr;
+	const Elf32_Shdr *section, *s;
```
Comment (lgirdwood) [inline rimage/elf.c:477]: This should be in rimage repo.

File: rimage/CMakeLists.txt [hunk 2]
```diff
@@ -32,6 +32,7 @@ add_executable(rimage
 	hash.c
 	pkcs1_5.c
 	manifest.c
+	ext_manifest.c
```
Comment (lgirdwood) [inline rimage/CMakeLists.txt:35]: ditto, rimage repo

File: rimage/ext_manifest.c [hunk 3]
```diff
@@ -0,0 +1,195 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
```
Comment (lgirdwood) [inline rimage/ext_manifest.c:2]: ditto

File: rimage/elf.c (hunk 1 above)
Comment (ktrzcinx) [reply rimage/elf.c:477]: https://github.com/thesofproject/rimage/pull/11

**Comment by lgirdwood** [comment]
@ktrzcinx looks like it's mainly rimage issue blocking this now ?

**Comment by ktrzcinx** [comment]
@lgirdwood it's only test PR, shouldn't be merge. I Close it

**Pull request closed without merging**